src/hotspot/share/opto/superword.cpp

```diff
@@ -43,14 +43,13 @@ SuperWord::SuperWord(const VLoopAnalyzer &vloop_analyzer) :
   _vloop_analyzer(vloop_analyzer),
   _vloop(vloop_analyzer.vloop()),
   _arena(mtCompiler),
-  _node_info(arena(), _vloop.estimated_body_length(), 0, SWNodeInfo::initial), // info needed per node
```

Comment:
Note: held the "alignment" info, all other fields were already removed in previous refactorings.